src/Widgets/EligibilityModal/__tests__/Cards.test.tsx

```diff
@@ -63,6 +64,25 @@ it('should display provided card logos', () => {
   expect(screen.queryByTestId('card-logo-visa')).not.toBeInTheDocument()
 })
 
+it('should display provided card logos with Desktop Modal', () => {
+  render(
+    <ResponsiveContext.Provider value={{ width: 801 }}>
```

Comment:
I think `800` should work since this is the min value
Also, if you are a cool person, you could take this opportunity to extract the `800`px constant somewhere, but it might be out of the scope of this PR.